### PR TITLE
fix(attachment page): Add missing attachment number in Appellate

### PR DESCRIPTION
**Pull request by ERosendo**
This PR resolves https://github.com/freelawproject/recap/issues/327.

This PR includes the following changes:

- This PR will update the if statement in the `getAttachmentNumberFromAnchor` method to avoid looking for an attachment number if the row has 3 or fewer children nodes. The current implementation uses the `<` which fails if the row that contains the doc links has exactly 3 children nodes ( The table from the docket report has exactly 3 data cell elements but we don't want to search the attachment number in that table).

- This PR adds the `attachment number` to the doc links URL if it's found on the page and also makes sure that the `caseId `is always added to the href attribute.

- We add a new if statement to check the queryparameters

### CODE REVIEW

File: src/appellate/appellate.js [hunk 1]
```diff
@@ -369,6 +369,11 @@ AppellateDelegate.prototype.onDocumentViewSubmit = function (event) {
 
   let title = document.querySelectorAll('strong')[1].innerHTML;
   let dataFromTitle = APPELLATE.parseReceiptPageTitle(title);
+
+  if (dataFromTitle.att_number == 0 && this.queryParameters.get('attNum')){
```
Comment (mlissner) [inline src/appellate/appellate.js:373]: Hm, a thought. If we're adding query parameters just for our own sake, should we namespace them? In other words, could we make any parameter we use start with recap, say? `recapAttNum`?

File: src/pacer.js [hunk 2]
```diff
@@ -644,7 +644,7 @@ let PACER = {
     //  attachment number of the document.
 
     let row = anchor.parentNode.parentNode;
-    if (row.childElementCount < 3) {
+    if (row.childElementCount <= 3) {
```
Comment (mlissner) [inline src/pacer.js:647]: Should we add a comment here to explain the different number of elements that are possible?

**Review by mlissner — COMMENTED** [review]
Couple quick suggestions for you, Eduardo!

File: src/appellate/appellate.js (hunk 1 above)
Comment (ERosendo) [reply src/appellate/appellate.js:373]: We can namespace the parameters. That's a great idea, I will change The set() method of the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface.

File: src/pacer.js (hunk 2 above)
Comment (ERosendo) [reply src/pacer.js:658]: Sure.

**Pull request merged**